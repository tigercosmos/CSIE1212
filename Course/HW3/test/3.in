3
.....
..OO.
..OO.
.XX..
.XX..

.....
.XOO.
..OO.
.X.X.
..X..

.....
..OX.
.O.O.
.XX..
.XO..
