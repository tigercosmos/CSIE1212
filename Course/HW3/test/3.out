O win
O win
Draw
